test/unittests/commands/xmlupload/test_ontology_lookup_models.py

```diff
@@ -166,5 +170,124 @@ def test_remove_prefixes_knora_properties() -> None:
+            "knora-api:objectType": {"@id": "knora-api:TextValue"},
+            "salsah-gui:guiElement": {"@id": "salsah-gui:SimpleText"},
+            "@id": "onto:hasSimpleText",
+        },
+        {
+            "rdfs:label": "Textarea",
+            "rdfs:subPropertyOf": {},
+            "knora-api:isResourceProperty": True,
+            "knora-api:objectType": {"@id": "knora-api:TextValue"},
+            "salsah-gui:guiElement": {"@id": "salsah-gui:Textarea"},
+            "@id": "onto:hasTextarea",
+        },
+        {
+            "rdfs:label": "Richtext",
+            "rdfs:subPropertyOf": {},
+            "knora-api:isResourceProperty": True,
+            "knora-api:objectType": {"@id": "knora-api:TextValue"},
+            "salsah-gui:guiElement": {"@id": "salsah-gui:Richtext"},
+            "@id": "onto:hasRichtext",
+        },
+        {
+            "rdfs:label": "Editor",
+            "rdfs:subPropertyOf": {},
+            "knora-api:isResourceProperty": True,
+            "knora-api:isLinkProperty": True,
+            "@type": "owl:ObjectProperty",
+            "knora-api:objectType": {"@id": "onto:Person"},
+            "salsah-gui:guiElement": {"@id": "salsah-gui:Searchbox"},
+            "@id": "onto:hasEditor",
+        },
+    ]
+    res = _get_all_text_value_properties_and_types_from_onto(test_json)
+    assert res == [
+        ("onto:hasSimpleText", "salsah-gui:SimpleText"),
+        ("onto:hasTextarea", "salsah-gui:Textarea"),
+        ("onto:hasRichtext", "salsah-gui:Richtext"),
+    ]
+
+
+def test_check_correct_val_type_cls() -> None:
+    test_dict = {
+        "knora-api:isResourceClass": True,
+        "rdfs:label": "Annotation",
+        "knora-api:canBeInstantiated": True,
+        "rdfs:subClassOf": [],
+        "rdfs:comment": "A generic class for representing annotations",
+        "@type": "owl:Class",
+        "@id": "knora-api:Annotation",
+    }
+    res = _check_correct_val_type(test_dict)
+    assert not res
+
+
+def test_check_correct_val_type_link_prop() -> None:
+    test_dict = {
+        "rdfs:label": "Editor",
+        "rdfs:subPropertyOf": {},
+        "knora-api:isResourceProperty": True,
+        "knora-api:isLinkProperty": True,
+        "@type": "owl:ObjectProperty",
+        "knora-api:objectType": {"@id": "onto:Person"},
+        "salsah-gui:guiElement": {"@id": "salsah-gui:Searchbox"},
+        "@id": "onto:hasEditor",
+    }
+    res = _check_correct_val_type(test_dict)
+    assert not res
+
+
+def test_check_correct_val_type_text_prop() -> None:
+    test_dict = {
+        "rdfs:label": "Textarea",
+        "rdfs:subPropertyOf": {},
+        "knora-api:isResourceProperty": True,
+        "knora-api:objectType": {"@id": "knora-api:TextValue"},
+        "salsah-gui:guiElement": {"@id": "salsah-gui:Textarea"},
+        "@id": "onto:hasTextarea",
+    }
+    res = _check_correct_val_type(test_dict)
+    assert res
+
+
+def test_make_text_value_property_gui() -> None:
+    test_li = [
```

Comment:
it took me a moment to understand what "li" is. The only abbreviations of list that are known to me are "lst" or "list" (unabbreviated)